Sources/DeepLearning/Layers/Recurrent.Swift

```diff
@@ -0,0 +1,192 @@
+// Copyright 2018 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Please rename the file: `Recurrent.Swift` -> `Recurrent.swift`.

Reply:
I don't think this comment has been addressed yet. `Pooling.Swift` and `Recurrent.Swift` currently still end with `.Swift` which has a capital `S`.

Reply:
This comment should've been addressed before the PR got merged. It triggered a weird failure during refactoring in #136.